src/test/regress/expected/typed_streams.out

```diff
@@ -4,11 +4,10 @@ SET search_path TO test_typed_streams,public;
 CREATE STREAM s0 (x integer, y integer);
 DROP TABLE s0;
 ERROR:  "s0" is not a table
-HINT:  Use DROP STREAM to remove a stream.
+HINT:  Use DROP FOREIGN TABLE to remove a foreign table.
```

Comment:
Playing around with this a bit more, I'm not sure if the quirkiness being introduced merits removing `RELKIND_STREAM`. We probably don't need `RTE_STREAM`, but `RELKIND_STREAM` seems to be the only way we can ensure good usability. But your call. 
```
usmanm=# CREATE STREAM s (x int);
CREATE STREAM
usmanm=# ALTER STREAM s ADD COLUMN y text;
ALTER STREAM
usmanm=# ALTER TABLE s ADD COLUMN z text;
ALTER TABLE
usmanm=# DROP FOREIGN TABLE s;
DROP FOREIGN TABLE
```